Docs/Workflows/AzureHosting/SwapAzureWebAppSlots.md

```diff
@@ -20,12 +20,14 @@ jobs:
       source-slot-name: Staging
       destination-slot-name: Production
       application-insights-resource-id: "Azure resource ID of the corresponding AI resource"
-      # Defaults to destination-slot-name if not set, used for adding git tags to the deployed commit.
+      # Defaults to 'production' if not set, used for adding git tags to the deployed commit.
       tag-prefix: production
-      # Defaults to source-slot-name if not set, used looking up the currently deployed commit by tag.
+      # Defaults to 'staging' if not set, used looking up the currently deployed commit by tag.
       swap-prefix: staging
       # The repository that hosts the code that will be swapped. This is necessary for adding tags and releases.
-      app-code-repo: Lombiq/Lombiq-GitHub-Actions 
+      app-code-repository: Lombiq/Lombiq-GitHub-Actions
+      # The branch, tag, or SHA to check out from the app-code-repository.
+      app-code-repository-ref: main
```

Comment:
Why is this necessary, actually? The commit to tag in the code repo should be found based on the `staging/latest` tag, as before.

Reply:
Yes this is exactly what we do, I implemented this based on your earlier comment https://github.com/Lombiq/GitHub-Actions/pull/375#discussion_r1694296342 I figured that there was maybe a certain flow that I missed that would be covered by adding this.
But perhaps I misinterpreted your comment.

Reply:
Yeah, I was talking about something else. I clarified there; I don't this parameter here is necessary.